.github/workflows/autoapprove.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow auto approves the PR generated by the bump_version
+# workflow, and moves the tag that was created in the PR's branch
+# to develop.
+
+name: github-actions auto-approve
+on: pull_request_target
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  approve-bot:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'semgrep-ci[bot]'}}
```

Comment:
Since this always triggers for `semgrep-ci[bot]`, let's just keep in mind we may need to narrow this to also require the PR title is `chore: release` or whatever if we run into additional cases we want this bot to make a PR here. Seems fine for now though.